pyproject.toml

```diff
@@ -1,22 +1,23 @@
 [tool.poetry]
 name = "automatic-udemy-course-enroller-get-paid-udemy-courses-for-free"
-version = "0.3"
+version = "1.0"
```

Comment:
think we need version 2.0 for this patch

Reply:
I was just bumping here to have the current release version. When we go to release again we just need to remember to bump it again

Reply:
Alright . Sounds good to me.